### DISCUSSION TITLE
Download cli and engine version 20.10.15 binary download

### DISCUSSION THREAD

**Discussion by amir734jj** [moby/moby#48572] [General]
Hi, I am wondering do we have a `https://mobyartifacts.azureedge.net` url for version 20.10.15 of moby engine and moby cli available?

This url doesn't work.
```
https://mobyartifacts.azureedge.net/moby/moby-engine/20.10.15+azure/windows/windows_amd64/moby-engine-20.10.15+azure-u3.amd64.zip
```

**Comment by cpuguy83** [comment]
These are artifacts for internal Microsoft consumption.
Also 20.10 is very old, really recommend you update.

Closing since this is not related to anything in this repo.

**Reply by amir734jj** [reply]
@cpuguy83 The reason I asked is because apparently `20.10.15` version is last supported version for windows server 2016.

Is it true?

**Reply by cpuguy83** [reply]
Hmm, I don't recall. I'd be shocked that we pulled support out after 20.10.15, there are much newer patch releases in the 20.10 branch.
